internal/v4/content_test.go

```diff
@@ -20,7 +20,7 @@ import (
 	"gopkg.in/juju/charmrepo.v2-unstable/csclient/params"
 
 	"gopkg.in/juju/charmstore.v5-unstable/internal/storetesting"
-	"gopkg.in/juju/charmstore.v5-unstable/internal/v4"
+	"gopkg.in/juju/charmstore.v5-unstable/internal/v5"
```

Comment:
It still feels strange for internal/v4_test to import internal/v5 instead of internal/v4, even if it is only for constants.